src/components/Capture/index.js

```diff
@@ -1,243 +1,36 @@
 import { h, Component } from 'preact'
-import { events } from 'onfido-sdk-core'
-import classNames from 'classnames'
-import randomId from '../utils/randomString'
-import { Uploader } from '../Uploader'
-import Camera from '../Camera'
-import Confirm from '../Confirm'
-import { FaceTitle } from '../Face'
-import { DocumentTitle } from '../Document'
-import isDesktop from '../utils/isDesktop'
-import DetectRTC from 'detectrtc'
-import style from './style.css'
-import { functionalSwitch, impurify } from '../utils'
-import { canvasToBase64Images } from '../utils/canvas.js'
-import { fileToBase64, isOfFileType, fileToLossyBase64Image } from '../utils/file.js'
+import Capture from './capture.js'
+import { impurify } from '../utils'
 
-const StatelessDocumentCapture = options =>
-  <Capture method='document' autoCapture={true} {...options} />
+const StatelessFrontDocumentCapture = options =>
```

Comment:
Ah, sorry - didn't mean to ignore this :)

So as far as I'm aware

```
export {
  a,
  b,
};

export c;

export default {
  d
}
```

just results in what is effectively

```
{
  a,
  b,
  c,
  default: {
    d
  },
}
```

And then `import { a } from ...` would work, `import { d } from ...` would fail, and `import def from ...; def.d` would work

Not sure if that's what you were asking or not?